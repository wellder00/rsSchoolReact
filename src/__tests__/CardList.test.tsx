// import { render, screen } from '@testing-library/react';
// import { MemoryRouter } from 'react-router-dom';
// import { CardBlock } from '../components/CardBlock';

// test('Verify that the component renders the specified number of cards', () => {
//   render(
//     <MemoryRouter initialEntries={["limit=10&offset=0&page=1"]}>
//       <CardBlock />
//     </MemoryRouter>
//   );
//   const cardWrapper = screen.getByTestId('card-wrapper');
//   expect(cardWrapper).toBeInTheDocument();
// });
