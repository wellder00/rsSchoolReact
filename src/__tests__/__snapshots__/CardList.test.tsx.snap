// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tests for the Card component > Check that an appropriate message is displayed if no cards are present 1`] = `
<DocumentFragment>
  <div
    class="_loading_d51639"
  >
    <div
      class="_loadingIcon_d51639"
      data-testid="loading-icon"
    >
      Loading...
    </div>
    <div
      class="_loader_d51639"
    >
      <div
        class="_loaderCircle_d51639"
      />
    </div>
  </div>
</DocumentFragment>
`;

exports[`Tests for the Card component > Verify that the component renders the specified number of cards; 1`] = `
<DocumentFragment>
  <div
    class="_loading_d51639"
  >
    <div
      class="_loadingIcon_d51639"
      data-testid="loading-icon"
    >
      Loading...
    </div>
    <div
      class="_loader_d51639"
    >
      <div
        class="_loaderCircle_d51639"
      />
    </div>
  </div>
</DocumentFragment>
`;
